scipy/linalg/tests/test_batch.py

```diff
@@ -286,3 +287,10 @@ def test_rsf2cs(self, dtype, rng):
         A = get_random((2, 3, 4, 4), dtype=dtype, rng=rng)
         T, Z = linalg.schur(A)
         self.batch_test(linalg.rsf2csf, (T, Z), n_out=2)
+
+    @pytest.mark.parametrize('dtype', floating)
+    def test_cholesky_banded(self, dtype, rng):
+        ab = get_random((5, 4, 3, 6), dtype=dtype, rng=rng)
+        ab[..., 0, 0] = 0
```

Comment:
Is this needed?

Reply:
No not strictly. I was getting a bit confused about the banded storage but I was under the impression if I didn't set anything to zero then the matrix it represents is just diagonal. I might have got that completely wrong though

Reply:
Reading the docs again I'm fairly sure we can just remove this line